Modules/UpendoDnnSimpleAuthProvider/Login.ascx.cs

```diff
@@ -403,17 +401,16 @@ private void OnLoginClick(object sender, EventArgs e)
                             // The entered verification code is invalid (does not exist in the database).
                             // Set the login status to indicate a login failure and display an error message.
                             loginStatus = UserLoginStatus.LOGIN_FAILURE;
-                            EventLogController.Instance.AddLog("User Login - Invalid Verification Code", "Username: " + userName, PortalController.Instance.GetCurrentSettings(), objUser.UserID, EventLogController.EventLogType.LOGIN_FAILURE);
-                            DotNetNuke.UI.Skins.Skin.AddModuleMessage(this, Localization.GetString("InvalidVerificationCode", this.LocalResourceFile), ModuleMessage.ModuleMessageType.RedError);
+                            DotNetNuke.UI.Skins.Skin.AddModuleMessage(this, Localization.GetString("AccessDenied", this.LocalResourceFile), ModuleMessage.ModuleMessageType.RedError);
+
                         }
                     }
                     else
                     {
                         // The verification code textbox is empty.
                         // Set the login status to indicate a login failure and display an error message.
                         loginStatus = UserLoginStatus.LOGIN_FAILURE;
-                        EventLogController.Instance.AddLog("User Login - Empty Verification Code", "Username: " + userName, PortalController.Instance.GetCurrentSettings(), objUser.UserID, EventLogController.EventLogType.LOGIN_FAILURE);
```

Comment:
@ArielBlanco1990 Why aren't you writing this event to the log anymore?

Reply:
Let's add the code back, but comment it out, just in case we need this one later.  (I don't normally request commented out code, but this one makes sense.)  :)